stream/cards.md

```diff
@@ -16,7 +16,30 @@ To facilitate users linking multiple cards, developers can add identifying key-v
 
 ### What Is Card Verification
 
-When linking a card, cardholders are required to complete a verification process to verify card ownership to ensure we protect them. The verification consists of the following steps:
+When linking a card, cardholders are required to complete one of the two available verification processes to verify card ownership to ensure we protect them.
+
+When linking a card, a property `verificationMethod` will identify what verification method that card is meant to follow - `3ds` or `microcharge`.
```

Comment:
I am not sure if adding this information will be helpful or if it will cause more confusion. Since it's an internal property, I don't think it's valuable to be stated here. We could simply say, 'When linking a card, two verification methods may be available: 3DS or Microcharge.' Wdyt?